Tone/component/filter/FeedbackCombFilter.ts

```diff
@@ -0,0 +1,81 @@
+import { Delay } from "../../core/context/Delay";
+import { Gain } from "../../core/context/Gain";
+import { Param } from "../../core/context/Param";
+import { InputNode, OutputNode, ToneAudioNode, ToneAudioNodeOptions } from "../../core/context/ToneAudioNode";
+import { NormalRange, Time } from "../../core/type/Units";
+import { optionsFromArguments } from "../../core/util/Defaults";
+import { RecursivePartial } from "../../core/util/Interface";
+
+export interface FeedbackCombFilterOptions extends ToneAudioNodeOptions {
+	delayTime: Time;
+	resonance: NormalRange;
+}
+
+/**
+ *  @class Comb filters are basic building blocks for physical modeling. Read more
+ *         about comb filters on [CCRMA's website](https://ccrma.stanford.edu/~jos/pasp/Feedback_Comb_Filters.html).
+ *
+ *  @extends {Tone.AudioNode}
+ *  @constructor
+ *  @param {Time|Object} [delayTime] The delay time of the filter.
+ *  @param {NormalRange=} resonance The amount of feedback the filter has.
+ */
+export class FeedbackCombFilter extends ToneAudioNode<ToneAudioNodeOptions> {
```

Comment:
The generic passed into the class should be the options object returned by `getDefaults` and passed into the constructor. This generic is used to in the `get` and `set` methods to define their options. 

If this line reads `
export class FeedbackCombFilter extends ToneAudioNode< FeedbackCombFilterOptions >` does it fix the issue that you were having with #539.